docker-compose.yml

```diff
@@ -1,4 +1,21 @@
-version: '3'
+version: '3.5'
 services:
   web:
-    build: ./
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - .:/app
+    ports:
+      - "5000:5000"
```

Comment:
Is it possible to use `FLASK_PORT` here too?

Reply:
I thought I did that... https://github.com/openstax/qualitas-server/commit/d0a7afbbb6d59a73e009031a3c4306d4d6888274

Reply:
Oh I see! I was on phone earlier.